node/subsystem/src/lib.rs

```diff
@@ -228,14 +228,24 @@ pub trait Subsystem<C: SubsystemContext> {
 /// types of messages. Used for tests or as a placeholder.
 pub struct DummySubsystem;
 
-impl<C: SubsystemContext> Subsystem<C> for DummySubsystem {
+impl<C: SubsystemContext> Subsystem<C> for DummySubsystem
+where
+	C::Message: std::fmt::Debug
+{
 	fn start(self, mut ctx: C) -> SpawnedSubsystem {
 		let future = Box::pin(async move {
 			loop {
 				match ctx.recv().await {
-					Ok(FromOverseer::Signal(OverseerSignal::Conclude)) => return Ok(()),
 					Err(_) => return Ok(()),
-					_ => continue,
+					Ok(FromOverseer::Signal(OverseerSignal::Conclude)) => return Ok(()),
+					Ok(overseer_msg) => {
+						log::debug!(
+							target: "dummy-subsystem",
```

Comment:
Fine by me, but if you are aware that a dummy subsystem is running, you probably already know that it swallows all the messages. So, you will never write `RUNTIME_LOG=dummy-subsystem=debug' :D

Reply:
yeah, I touched this above: in my case I ran with `RUST_LOG=debug` and the amount of output was bearable. I added `target` not for filtering, but just to make it (a little bit more) clear where this log statement comes from.

Reply:
Yeah